pdns/lua-recursor4-ffi.hh

```diff
@@ -89,4 +110,18 @@ extern "C"
   void pdns_ffi_param_set_padding_disabled(pdns_ffi_param_t* ref, bool disabled) __attribute__((visibility("default")));
   void pdns_ffi_param_add_meta_single_string_kv(pdns_ffi_param_t* ref, const char* key, const char* val) __attribute__((visibility("default")));
   void pdns_ffi_param_add_meta_single_int64_kv(pdns_ffi_param_t* ref, const char* key, int64_t val) __attribute__((visibility("default")));
+
+  typedef struct pdns_postresolve_ffi_handle pdns_postresolve_ffi_handle_t;
+
+  const char* pdns_postresolve_ffi_handle_get_qname(pdns_postresolve_ffi_handle_t* ref) __attribute__((visibility("default")));
+  uint16_t pdns_postresolve_ffi_handle_get_qtype(const pdns_postresolve_ffi_handle_t* ref) __attribute__((visibility("default")));
+  uint16_t pdns_postresolve_ffi_handle_get_rcode(const pdns_postresolve_ffi_handle_t* ref) __attribute__((visibility("default")));
```

Comment:
A `set_rcode` function would be useful too.

Reply:
Likely setting the policy kind is what you want in many cases. But if you are going to build an answer by hand, you would need to set the rcode, yes.

Reply:
We are currently not using those fields, but we could if they are useful. It would still be useful to be able to set any rcode we want, just as we have the ability to modify the rrsets anyway we want.

Reply:
Latest version has `pdns_postresolve_ffi_handle_set_rcode`